roles/nut/templates/ups.conf.j2

```diff
@@ -0,0 +1,8 @@
+# {{ ansible_managed }}
+
+maxretry = 3
+
+[ups-{{ ansible_hostname[0]|upper }}]
+	driver = usbhid-ups
+	port = auto
+	desc = "Onduleur du Bâtiment {{ ansible_hostname[0]|upper }}"
```

Comment:
I just realized that we have two racks in D building, so the first hostname letter isn't enough :grimacing: I think we will need to define a host variable for it (also we already have a `server_location` variable for the `monitoring` role, we might be able to reuse (part of) it with YAML tricks)

Reply:
Resolved with [fefab0a](https://github.com/rezoleo/ansible-playbooks/pull/6/commits/fefab0ae88a104176dabcfbb748fd0f79651b91a)